index.js

```diff
@@ -16,35 +16,190 @@ const forbiddenModules = [
   // want to restrict create-react-class usage within our modules
   'create-react-class',
   // want to prevent incorrect imports of moment
-  { name: 'moment/moment', message: "Import only from 'moment' instead." },
+  {name: 'moment/moment', message: "Import only from 'moment' instead."},
 ];
 
 module.exports = {
   parser: 'babel-eslint',
-  extends: [
-    'eslint-config-airbnb',
-  ].map(require.resolve).concat([
-    // Some helpful rules that will prevent bugs
-    'plugin:unicorn/recommended',
-    // Use native JS instead of lodash
-    'plugin:you-dont-need-lodash-underscore/compatible',
-  ]),
+  parserOptions: {
+    ecmaVersion: 11,
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
   env: {
     browser: true,
-    node: true,
+    // we are using optional chainig and nullish coalescing
+    es2020: true,
+    // do not warn about "it", "describe", "assert", etc. from showing as undefined globals
+    jest: true,
+    'jest/globals': true,
+  },
+  globals: {
+    // these globals help IDEs from displaying these as undefined
+    jest: true,
+    localStorage: true,
+    process: false,
+    Promise: true,
+  },
+  settings: {
+    // the src folder can be used as the resource root for imports
+    'import/resolver': {
+      node: {
+        paths: ['src'],
+      },
+    },
   },
+  extends: ['eslint-config-airbnb']
+    .map(require.resolve)
+    .concat([
+      'eslint:recommended',
+      'plugin:unicorn/recommended',
+      'plugin:react/recommended',
+      'plugin:jest/recommended',
```

Comment:
I think the jest plugin is getting confused by our `mock/scenarios.js` file that is in all our modules.
```
/Users/bsmcinty/workspace/lending/axp-non-card-lending-merchant-financing-payment-schedule/mock/scenarios.js
  15:3  warning  Test has no assertions          jest/expect-expect
  19:3  warning  Test has no assertions          jest/expect-expect
  24:3  warning  Test has no assertions          jest/expect-expect
  29:3  warning  Test has no assertions          jest/expect-expect
  34:3  warning  Test has no assertions          jest/expect-expect
  39:3  warning  Test has no assertions          jest/expect-expect
  47:1  error    Do not export from a test file  jest/no-export
```
For this file:
https://stash.aexp.com/stash/projects/AIM500001188/repos/axp-non-card-lending-merchant-financing-payment-schedule/browse/mock/scenarios.js

We need to avoid that error somehow.

Reply:
If a test doesn't expect an assertion, is it really a test? 😉